README.md

```diff
@@ -1,22 +1,24 @@
 # Pitchfork
 
-Pitchfork is a framework forming the basis for [Trident](https://trident.li) and other tools build upon this framework.
+Pitchfork is a framework forming the basis for [Trident](https://trident.li)
+and other tools build upon this framework.
 
 ## Testing
 
-We include tests using the go [testing](https://golang.org/pkg/testing/) framework.
+Tests are included using the go [testing](https://golang.org/pkg/testing/)
+framework.
 
 ### Running Tests
 
-Running tests requires that two environment variables are set:
+Running tests requires two environment variables to be are set:
 ```
 export PITCHFORK_TOOLNAME=trident
-export PITCHFORK_CONFROOT=/Users/jeroen/git/trident/tconf/
+export PITCHFORK_CONFROOT=/Users/${USER}/git/trident/tconf/
```

Comment:
sure fixed.

also, please point me to a sample, usable-for-testing, config file I'd put in trident/tconf ... without that, we are in 'horrible dumpster fire' land again.